.sbtrc

```diff
@@ -1 +1 @@
-alias boot = ;reload ;project kittens ;iflast shell
+alias boot = ;reload ;project / ;iflast shell
```

Comment:
No idea what this is for

Reply:
@joroKr21 This `.sbtrc` file? Or the change I made here?

This was a very strange setup and if you don't know what it is I think we should just remove it.

Reply:
Yes the `.sbtrc` file